.github/workflows/python-test.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             # ########################
             ## lal extras fetch method
             #git lfs pull
-            # wget https://git.ligo.org/lscsoft/lalsuite-extra/-/raw/master/data/lalsimulation/SEOBNRv4ROM_v2.0.hdf5 -O SEOBNRv4ROM_v2.0.hdf5
+            wget https://git.ligo.org/lscsoft/lalsuite-extra/-/raw/master/data/lalsimulation/SEOBNRv4ROM_v2.0.hdf5 -O SEOBNRv4ROM_v2.0.hdf5
```

Comment:
Pls remind me, what are we using `SEOBNRv4ROM` for? Can we replace it with `SEOBNRv4` which is a bit slower but probably still faster than downloading this file.

Reply:
Also, shorten the path a few lines above (line 49):
```
export LAL_DATA_PATH="${PWD}/test_data/lalsuite/lalsuite-extra/data/lalsimulation"
```
to 
```
export LAL_DATA_PATH="${PWD}/lalsuite-extra/data/lalsimulation"
```

Reply:
Finally, how is [this test](https://github.com/gwnrtools/nr-catalog-tools/actions/runs/5023480264/jobs/9008119973) failing even after you enabled the download of the required file?